core/sds-aspect-meta-model-java/src/main/java/io/openmanufacturing/sds/metamodel/EntityInstance.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2022 Robert Bosch Manufacturing Solutions GmbH
+ *
+ * See the AUTHORS file(s) distributed with this work for additional
+ * information regarding authorship.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+package io.openmanufacturing.sds.metamodel;
+
+import java.util.Map;
+
+public interface EntityInstance extends Base, IsDescribed, Value, Comparable<EntityInstance> {
+   Map<Property, Value> getAssertions();
```

Comment:
I was thinking about assertions as in "setting a value". I would not call it `getValues`, because each entry is not only the entry, but asserts a value to a given Property. I'm open to change this if we find a better name.